src/Microsoft.Dnx.Tooling/Restore/RestoreCommand.cs

```diff
@@ -199,7 +207,7 @@ public RestoreCommand(FrameworkName fallbackFramework)
             }
         }
 
-        private async Task<bool> RestoreForProject(string projectJsonPath, string rootDirectory, string packagesDirectory, SummaryContext summary)
+        private async Task<bool> RestoreForProject(string projectJsonPath, string rootDirectory, string packagesDirectory, IList<IWalkProvider> remoteProviders, SummaryContext summary)
```

Comment:
side note - this function has almost 300 lines. I saw more functions that are as big. We need to start thinking about breaking this huge functions up.